src/GeositeFramework/App_Data/regionSchema.json

```diff
@@ -31,6 +31,8 @@
                     "text": {"type": "string", "required": true},
                     "url": {"type": "string", "required": true},
                     "popup": {"type": "boolean", "required": false},
+                    "launchpad": {"type": "boolean", "required": false},
+                    "launchpadId": {"type": "integer", "required": false},
```

Comment:
Consider using a string for id, it might be a little easier for the configur-er to match the launchmap id as a name to the link in the config, rather than an arbitrary number.